### DISCUSSION TITLE
Why is Service_Code (Place of Service Code) not an 'array'?

### DISCUSSION THREAD

**Discussion by hhCambia** [CMSgov/price-transparency-guide#112] [Q&A] [answered]
We are concerned about the size of the final file. 

Because Service_Code is not an array, we will have to repeat the fee for all of the billing codes for about 50 places of service.  

In the majority of  our provider agreements we follow CMS's approach for reimbursement. Therefore our reimbursement rate is the same for each place of service code that is assigned by CMS as 'facility'. Likewise the rate is the same for each place of service code that is assigned as 'non-facility'. Since we are following this methodology, we would expect the 'place of service' be defined as an array. 

Doing so would reduce the 'place of service' to 2 arrays in our case therefore reducing file size significantly. 

We realize that there are similar threads on this topic already (for example, https://github.com/CMSgov/price-transparency-guide/discussions/99). But we are raising this issue again hoping to advance the conversation.

**Comment by ryan-thomas-peterson** [comment, marked as answer]
@hhCambia , looks like this one is addressed in the latest schema update and is now an array

**Comment by shaselton-usds** [comment]
@hhCambia Just incase you wanted the specific commit -- you can find it here: https://github.com/CMSgov/price-transparency-guide/commit/ad66fa17e464ed43fb58d3c6680808819d4b2c7b
